keras_cv/layers/preprocessing/solarization_test.py

```diff
@@ -36,14 +36,21 @@ def test_range_0_to_255(self):
                 dtype=tf.uint8,
             )
 
-    @staticmethod
-    def _test_input_output(layer, input_value, expected_value, dtype):
-        dummy_input = tf.ones(shape=(2, 224, 224, 3), dtype=dtype) * input_value
-        expected_output = tf.ones(shape=(2, 224, 224, 3), dtype=dtype) * expected_value
+    def test_solarization_with_addition(self):
+        solarization = Solarization(addition=10.0)
 
-        output = layer(dummy_input)
+        test_parameters = [
```

Comment:
I think we should consider using absl parameterized to config the test case. It will generate one test case per param setting. In the current setting, the failing test in front will block the following test cases.

https://github.com/abseil/abseil-py/blob/3e4749af698201224165a98930c6845d8c102e0d/absl/testing/parameterized.py#L73

Reply:
I will second for parametized test cases, useful in refactoring other test cases as well

Reply:
done - thanks for the suggestion, this is great.